defaults/main.yml

```diff
@@ -5,6 +5,8 @@
 consul_debug: false
 virtualenv: "{{ lookup('env', 'VIRTUAL_ENV') | default('') }}"
 consul_install_dependencies: true
+consul_rolling_restart: false
+consul_rolling_restart_delay_sec: 5
```

Comment:
Let's move these variables to the "Install Method" section (on line 37), I think they make more sense there.